unity/unity.c

```diff
@@ -1823,7 +1823,10 @@ void UnityIgnore(const char* msg, const UNITY_LINE_TYPE line)
         UNITY_OUTPUT_CHAR(' ');
         UnityPrint(msg);
     }
-    UNITY_IGNORE_AND_BAIL;
+
+	UNITY_PRINT_EOL();
+
+	UNITY_IGNORE_AND_BAIL;
```

Comment:
User 4 spaces instead of tabs. We would like to keep the same formatting as it is external library. You can ignore github action formatter comments here.